packages/core/src/ui/Base/CustomLink.tsx

```diff
@@ -1,22 +1,47 @@
 import Link from "next/link.js";
+import { Tooltip } from "../Tooltip";
+
+type Config = {
+  showLinkPreviews: boolean;
+};
 
 interface Props {
   href?: string;
+  data: any;
+  usehook: any;
+  config: Config;
   [x: string]: unknown;
 }
 
-export const CustomLink: React.FC<Props> = ({ href, ...rest }) => {
+export const CustomLink: React.FC<Props> = ({
+  config,
+  data,
+  usehook,
+  ...props
+}) => {
+  const { href } = props;
```

Comment:
I had done this initially and it was one of the problems I got that the tooltip content was not rendering. Tried a bit again on this but still issues, so left it at that. Maybe we can come back to this later if needed ?

Reply:
Ok, sure. Just please leave a comment above this line, so that we remember, that changing it will cause an error. I know myself and in the future, if I forget about it my perfectionism will kick in and I will try to add it to the first destructuring for sure 😝